consensus/Cargo.toml

```diff
@@ -53,6 +53,11 @@ schemadb = { path = "../storage/schemadb" }
 short-hex-str = { path = "../crates/short-hex-str" }
 storage-interface = { path = "../storage/storage-interface" }
 
+[dependencies.rocksdb]
+version = "0.17.0"
+default-features = false
+features = ["lz4"]
+
```

Comment:
it'll be nice if we expose those needed from this from schemadb and avoid depending on rocksdb everywhere.